internal/topo/context/func_context.go

```diff
@@ -56,12 +56,14 @@ func (c *DefaultFuncContext) GetFuncId() int {
 	return c.funcId
 }
 
-func (c *DefaultFuncContext) GetConnection(connectSelector string) (interface{}, error) {
-	return connection.GetConnection(connectSelector)
+func (c *DefaultFuncContext) GetConnection(clientType string, props map[string]interface{}) (interface{}, error) {
+	reqId := fmt.Sprintf("%s_%s_%d", c.GetRuleId(), c.GetOpId(), c.GetInstanceId())
```

Comment:
just delegate to the c.StreamContext like other overwritten functions should be fine